cpp/tests/test_poly_geometric_path.cpp

```diff
@@ -1,6 +1,8 @@
 #include "gtest/gtest.h"
 #include <Eigen/Dense>
 
+#include <Eigen/src/Core/util/Constants.h>
+#include <gmock/gmock-matchers.h>
```

Comment:
```suggestion
```

I think you don't need this two headers. With Eigen, you should never include stuff in `Eigen/src` yourself. I think the second is not used below.